### ISSUE TITLE
The DECOLLE loss seems not to be applied in the training process

### ISSUE THREAD

**Issue by zjuls** [nmi-lab/decolle-public#22]
In the train_snn_bptt.py, the loss function is set as the CrossEntrophy function instead of the decolle loss in line 79. So how is the layer-wise decolle loss function applied during training?

**Comment by eneftci** [comment]
Note that cross entropy doesn't mean the losses are global, as cross entropy could also be used for local losses. But you are right that scripts under test use BPTT. This is because test is for testing the network in the best case training scenario, hence it uses BPTT. The decolle scripts (with local learning) are under scripts/.
